frontend/src/components/home-page/components/filter-dialog.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'hooks/hooks';
+import Button from '../../common/button/button';
+import Checkbox from '../../common/checkbox/checkbox';
+import styles from '../styles.module.scss';
+
+const FilterDialog: React.FC = () => {
+
+  const checkBoxesCount = 22;
+
+  const [isChecked, setChecked] = useState(false);
+
+  const onCheckBoxChange = ():void => {
+    setChecked(!isChecked);
+  };
+
+  return (
+    <div className={styles.filterDialog}>
+      <div className={styles.titleBlock}>
+        <span className={styles.filter}>Filter</span>
+        <span className={styles.clear} onClick={():void => setChecked(false)}>Clear All</span>
+      </div>
+      <div className={styles.filtersBlock}>
+        <ul>
+          {[...Array(checkBoxesCount)].map((item, index) => <li key={index}>
```

Comment:
```suggestion
          {[new Array(checkBoxesCount)].map((item, index) => <li key={index}>
```

Reply:
I have tried this syntax "[new Array(checkBoxesCount)]" and the element was rendered only once. I have looked through this article and found an advice to use the [...Array(n)] syntax for such cases https://simplernerd.com/jsx-repeat-n-times

Reply:
hot sorry, it should be `[...new Array(checkBoxesCount)]` (just with new)

Reply:
thanks) I had another opportunity to read about arrays